### PR TITLE
Store core handle instead of core_id in ocf_request

### PR DESCRIPTION
**Pull request by robertbaldyga**

### CODE REVIEW

File: src/engine/engine_bf.c [hunk 1]
```diff
@@ -51,25 +51,25 @@ static void _ocf_backfill_complete(struct ocf_request *req, int error)
...
+	ctx_data_free(cache->owner, req->data);
+	req->data = NULL;
+
+	if (req->error) {
+		env_atomic_inc(&req->core->counters->cache_errors.write);
+		ocf_engine_invalidate(req);
+	} else {
+		ocf_req_unlock(req);
+
+		/* always free the request at the last point
```
Comment (arutk) [inline src/engine/engine_bf.c]: Can you move more text to this line to fill 80 char width?

File: src/engine/engine_common.c [hunk 2]
```diff
@@ -24,7 +24,7 @@ void ocf_engine_error(struct ocf_request *req,
 	if (stop_cache)
 		env_bit_clear(ocf_cache_state_running, &cache->cache_state);
 
-	ocf_core_log(&cache->core[req->core_id], log_err,
+	ocf_core_log(cache->core, log_err,
```
Comment (arutk) [inline src/engine/engine_common.c]: The first argument should be "req->core"

Comment (robertbaldyga) [reply src/engine/engine_common.c]: Done!

File: src/engine/engine_bf.c (hunk 1 above)
Comment (robertbaldyga) [reply src/engine/engine_bf.c]: Done. After some rephrasing it actually fits into 80 chars.

**Pull request merged**